src/qibotn/backends/quimb.py

```diff
@@ -24,12 +24,40 @@ def __init__(self, runcard):
             else:
                 raise TypeError("MPS_enabled has an unexpected type")
 
+            global TEBD_enabled
+            global TEBD_option
```

Comment:
Here it is pretty simple: never use `global` anywhere.

There are special cases in which you might want to abuse the module system and make a smart use of the `global` keyword.
Well, even in those cases you could apply the principle
> Special cases aren't special enough to break the rules.
> https://peps.python.org/pep-0020/

However, especially if you're not redesigning the whole library, please always avoid the use of `global` statements.